src/main/java/com/easypost/model/Shipment.java

```diff
@@ -548,6 +550,56 @@ public void setTaxIdentifiers(final List<TaxIdentifier> taxIdentifiers) {
         this.taxIdentifiers = taxIdentifiers;
     }
 
+    /**
+     * Get the carriers accounts of this shipment.
+     *
+     * Get the carrier accounts list.
+     *
+     * @return List of carrier accounts.
+     */
+    public List<CarrierAccount> getCarrierAccounts() {
+        return carrierAccounts;
+    }
+
+    /**
+<<<<<<< HEAD
+     * Set the carrier accounts list.
+=======
+     * Set the carriers accounts of this shipment.
+>>>>>>> 5dc59b5 (fixed typo for getter function and add description to each function in docstring.)
```

Comment:
It looks like you did not rebase correctly. Please fix this before we review again.

Reply:
corrected